app/src/main/java/org/fossasia/susi/ai/activities/LoginActivity.java

```diff
@@ -31,10 +38,13 @@
 
     @BindView(R.id.email)
     TextInputLayout email;
+    @BindView(R.id.email_input)
```

Comment:
@the-dagger @daminisatya will making the Autocompletetextview private solve the issue correctly?